.github/workflows/android.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       with:
         distribution: temurin
         java-version: 17.0.6
-        cache: gradle
+        # cache: gradle
```

Comment:
Is this intended?

Reply:
yes, that step only takes like 10 seconds without the cache, and uses 400mb of cache

Reply:
Let's add a comment why then

Reply:
actually i put it back, it affects the ability of ccache to work properly.